Sources/MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -79,27 +79,30 @@ public extension VisualInstruction {
             let container = try decoder.container(keyedBy: CodingKeys.self)
             let kind = try container.decode(Kind.self, forKey: .kind)
             
-            let text = try container.decode(String.self, forKey: .text)
+            let text = try container.decodeIfPresent(String.self, forKey: .text)
```

Comment:
`text` is documented as being required in the API response, so we should be calling `decode(_:forKey:)` here.

Reply:
Ah, right, lane components don’t have `text`. That’s an omission in the documentation.